scripts/tools/publish_model_registry_release.py

```
@@ -0,0 +1,394 @@
+def main(argv: Sequence[str] | None = None) -> int:
+    """Run the model registry publication workflow."""
+    parser = _build_parser()
+    args = parser.parse_args(list(argv) if argv is not None else None)
+    args.registry_path = args.registry_path.resolve()
+    args.cache_dir = args.cache_dir.resolve()
+    registry_data = _load_registry_data(args.registry_path)
+    plan = _stage_assets(args, registry_data)
+
+    if args.execute_upload:
+        if args.create_release:
+            subprocess.run(plan["create_release_command"], check=True)
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)